clients/src/main/java/org/apache/kafka/clients/admin/NewTopic.java

```diff
@@ -31,12 +31,24 @@
  * A new topic to be created via {@link AdminClient#createTopics(Collection)}.
  */
 public class NewTopic {
+
+    public static final int DEFAULT_PARTITIONS = -1;
+    public static final short DEFAULT_REPLICATION_FACTOR = -1;
```

Comment:
@agavra I don't think these should be public API.

Reply:
sounds good - I thought it would be a nice temp workaround for not having a builder (though an 'advanced' user can always just pass in `-1` in the main constructor